models/dodo/arbitrum/dodo_arbitrum_schema.yml

```
@@ -165,7 +165,7 @@ models:
             - trace_address
       - check_dex_aggregator_seed:
           blockchain: arbitrum
-          project: DODO X
+          project: DODO
```

Since `project` is varchar typed, it is case-sensitive.
```suggestion
          project: dodo
```